wsapi.go

```diff
@@ -379,6 +382,22 @@ func (s *Session) UpdateStatus(idle int, game string) (err error) {
 	return s.UpdateStreamingStatus(idle, game, "")
 }
 
+//UpdateListeningStatus is used to set the user to "Listening to..."
+//If game!="" then set to what user is listening to
+//Else, set user to active and no game.
+func (s *Session) UpdateListeningStatus(idle int, game string) (err error) {
+	gameType := GameTypeListening
+	usd := UpdateStatusData{
+		Status: "online",
+	}
+	usd.Game = &Game{
+		Name: game,
+		Type: gameType,
```

Comment:
This probably reads a bit better as:


return s.UpdateStatusComplex(UpdateStatusData{
  Status: "online",
  Game: &Game {
    Name: game,
    Type: GameTypeListening,
    URL: "",
  }
})